src/server/utils/safeRequest.js

```diff
@@ -25,11 +27,8 @@ function pick(request, safeKeys) {
 }
 
 const requestAllowList = [
-  'acceptsLanguages',
-  'baseUrl',
-  'forwarded',
+  // 'forwarded', // not in use (?) need to find a fastify equivalent
```

Comment:
if this is a TODO can we mark it as such? and then, can we clear this TODO before merge?

Reply:
thanks, this can be enabled again since `forwardedHeaderParser` was migrated already and injects the `forwarded` key into the `request` object